### PR TITLE
Fix the problem of long connection for more than 30 seconds and disconnection of the server

### PR DESCRIPTION
**Pull request by 235832289**
The problem of long connection for more than 30 seconds and disconnection of the server
#770

### CODE REVIEW

**Comment by fatedier** [comment]
You'd better not use chinese in commit log, it's not friendly for others.

File: cmd/frps/root.go [hunk 1]
```diff
@@ -37,29 +37,30 @@ var (
...
+	logLevel              string
+	logMaxDays            int64
+	token                 string
+	authTimeout           int64
+	subDomainHost         string
+	tcpMux                bool
+	allowPorts            string
+	maxPoolCount          int64
+	maxPortsPerClient     int64
+	ResponseHeaderTimeout int64
```
Comment (fatedier) [inline cmd/frps/root.go]: Use lowercase letter at the beginning and the word `ResponseHeaderTimeout` is not clear for others to figure out where it's used.
`vhost_http_timeout` is better.

File: cmd/frps/root.go [hunk 2]
```diff
@@ -85,6 +86,7 @@ func init() {
 	rootCmd.PersistentFlags().Int64VarP(&authTimeout, "auth_timeout", "", 900, "auth timeout")
 	rootCmd.PersistentFlags().StringVarP(&subDomainHost, "subdomain_host", "", "", "subdomain host")
 	rootCmd.PersistentFlags().Int64VarP(&maxPortsPerClient, "max_ports_per_client", "", 0, "max ports per client")
+	rootCmd.PersistentFlags().Int64VarP(&ResponseHeaderTimeout, "Response_Header_Timeout", "", 30, "Response Header Timeout")
```
Comment (fatedier) [inline cmd/frps/root.go]: Use lowercase letters.

File: conf/frps_full.ini [hunk 3]
```diff
@@ -66,3 +66,6 @@ subdomain_host = frps.com
 
 # if tcp stream multiplexing is used, default is true
 tcp_mux = true
+
+# if Response Header Timeout more the 30s ,fix the pars .
+ResponseHeaderTimeout = 30
```
Comment (fatedier) [inline conf/frps_full.ini]: Use lowercase letters and underline.

File: cmd/frps/root.go (hunk 1 above)
Comment (235832289) [reply cmd/frps/root.go]: OK, I've changed it

File: cmd/frps/root.go (hunk 2 above)
Comment (235832289) [reply cmd/frps/root.go]: OK, I've changed it

File: conf/frps_full.ini (hunk 3 above)
Comment (235832289) [reply conf/frps_full.ini]: OK, I've changed it

**Comment by fatedier** [comment]
Please rebase all commit log into one without chinese.

**Pull request closed without merging**